boards/nordic/nrf54lm20apdk/board.cmake

```diff
@@ -1,7 +1,11 @@
 # Copyright (c) 2024 Nordic Semiconductor ASA
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 
-board_runner_args(jlink "--device=cortex-m33" "--speed=4000")
+if(CONFIG_BOARD_NRF54LM20APDK_NRF54LM20A_CPUAPP)
+  board_runner_args(jlink "--device=cortex-m33" "--speed=4000")
+elseif(CONFIG_BOARD_NRF54LM20APDK_NRF54LM20A_CPUFLPR)
```

Comment:
can use SOC symbols here (optional but preferred) so that customers can copy the board and it will just work